Bottom: fc878517f43af5ee9b339295005b86a6c08e5e39
Top:    1c0d2f49091c0c9ebf1e7eadf40bc7094d0978de
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-28 21:28:20 +0100

Refresh of parse_y-y-don-t-bother-setting

---

diff --git a/src/parse_y.y b/src/parse_y.y
index 86ed589..ed4d183 100644
--- a/src/parse_y.y
+++ b/src/parse_y.y
@@ -226,8 +226,6 @@ parsepcb
 		    PCBType *pcb_save = PCB;
 		    ElementType *e;
 
-		    LayerFlag[0] = true;
-		    LayerFlag[1] = true;
 		    yyData->LayerN = 2;
 
 		    CreateNewPCBPost (yyPCB, 0);
